benchmark/go.mod

```diff
@@ -0,0 +1,35 @@
+module k8s.io/perf-tests/benchmark
+
+go 1.15
+
+require (
+	github.com/aclements/go-moremath v0.0.0-20190830160640-d16893ddf098 // indirect
```

Comment:
Why do we remove all of these from vendor/ ?

In other packages already using go modules (like clusterloader2/ vendor seem to be fully propagated...)

Reply:
With this all the packages will be stored in GOPATH rather than in vendor. Like how perfdash, util-images does, I don't know why we still have a vendor folder in clusterloader2 when we have already migrated to go modules.

Reply:
That's actually desired to have vendor - we do exactly the same in the main repo:
https://github.com/kubernetes/kubernetes/tree/master/vendor

Reply:
Got it I added the `go mod vendor`